storage/storage_setup.py

```diff
@@ -5,17 +5,31 @@
 #User arguments should contain the following content in that order: ip address, username, password, keyspace
 
 def cluster_setup(ip_address, username, password, keyspace):
+<<<<<<< HEAD
```

Comment:
looks like you have some merge conflict code you still need to clean up

Reply:
this file keeps creating merging files even though i havent touched it in weeks....anyway, i fixed the conflict i hope